### PR TITLE
Add: django-content-settings

### PR DESCRIPTION
**Pull request by oduvan**
## Project Information

1. **Project Name:** django-content-settings
2. **Project URL:** https://github.com/occipital/django-content-settings
3. **Description:** _offering users the ability to easily create and manage editable variables directly from the Django admin panel. What sets this module apart is its ability to handle variables of any type without restricting their complexity. Thanks to an integrated caching system, these variables can be used efficiently in code, irrespective of their complexity._

## Criteria

Please answer the following questions about the project you are submitting. This will help us evaluate if the project should be included in the Awesome Django list.

1. **Is the project new?**
   - [x] Yes
   - [ ] No

2. **How long has the project been maintained?**
   for more than a year, but for a long time it was part of an internal project, and recently we decided to move it in open source as it becomes part of more and more projects

3. **How many releases has it had if it's a library or package?**
   _ As an independent open source projects, it doesn't have many releases yet_

4. **Are you the author or are you submitting the project on behalf of a company?**
   - [x] I am the author
   - [x] I am submitting on behalf of a company
   - [ ] Other (please specify)

5. **What makes it awesome?**
   the way you define settings, the flexibility you have to customize settings to your needs, admin panel for easy and flexible catalog (which includes changing custom changing history), massive changing settings in one transaction, advanced caching system, typing version, and the most important it is very simple

## Additional Information

Please provide any additional information that you believe is relevant to the project or its evaluation for inclusion in the Awesome Django list. This might include information about the project's documentation, test coverage, community support, or any unique features that set it apart from other Django projects.

I think those questions have been answered in the fifth question. full documentation link you can find here https://django-content-settings.readthedocs.io/en/latest/. 89% test coverage

### CODE REVIEW

**Comment by oduvan** [comment]
I add test coverage.

**Comment by jefftriplett** [comment]
Thank you for the submission. We'll run this through the newsletter and see if it picks up some steam.

**Comment by oduvan** [comment]
0.9 version released with a lot of improvements and also a "short" video with dive into 

https://youtu.be/RNYmvv_G5zs?si=dz9f2RGtrldFNYKq

File: README.md [hunk 1]
```diff
@@ -146,6 +146,7 @@ _For a complete listing of all available packages, see [Django Packages](https:/
 - [environs](https://github.com/sloria/environs) - Simplified environment variable parsing that comes with a [Django helper](https://github.com/sloria/environs#usage-with-django) that installs additional packages.
 <!--lint enable double-link-->
 - [django-classy-settings](https://github.com/funkybob/django-classy-settings) - Class based settings to keep your environments in order, with easy access to typed environment variables.
+- [django-content-settings](https://django-content-settings.readthedocs.io/en/latest/) - easily create and manage editable typed variables directly from the Django admin panel
```
Comment (jefftriplett) [inline README.md:149]:
```suggestion
- [django-content-settings](https://django-content-settings.readthedocs.io/en/latest/) - easily create and manage editable typed variables directly from the Django admin panel.
```

**Comment by jefftriplett** [comment]
Thank you for your submission and patience. I approved it because I do think it could be really handy for a few niche cases. Well done!

**Comment by oduvan** [comment]
@jefftriplett thanks again for approving that. I just want you guys to know I did some significant progress since the first submit.

**Pull request merged**